src/test/java/io/zipcoder/interfaces/StudentsTest.java

```diff
@@ -0,0 +1,27 @@
+package io.zipcoder.interfaces;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+public class StudentsTest {
+
+    @Test
+    public void getInstance() throws Exception {
```

Comment:
`Exception` is too abstract.
You avoid using such an abstract class in a `throws` signature at all costs.